test/remote/gateways/remote_orbital_test.rb

```diff
@@ -746,7 +746,7 @@ def test_credit_purchase_with_address_responds_with_name
 
   def test_truncates_and_removes_accents_from_name
     truncated_name = 'Jose Maria Lopez Garc'
-    credit_card = credit_card('4556761029983886', first_name: 'José María', last_name: 'López García')
+    credit_card = credit_card('4556761029983886', first_name: ':) José María', last_name: 'López García')
```

Comment:
:) passes without issues even if you remove the `gsub(/[^a-zA-Z ]/, ' ')`

Reply:
Oh it looks like the sandbox doesn't work the same way that in production. Even sending emojies and accents doesn't cause any issues.

Reply:
I updated it to remove the emojis and emoticons in the consumer name